go.mod

```diff
@@ -25,10 +25,12 @@ require (
 	github.com/pterm/pterm v0.12.54
 	github.com/qeesung/image2ascii v1.0.1
 	github.com/santhosh-tekuri/jsonschema/v5 v5.2.0
+	github.com/spectrocloud-labs/herd v0.3.0
```

Comment:
Not sure why herd is pulled, I can't see it used anywhere?

Reply:
no idea to be honest.. good point, I've just run go mod tidy..

Reply:
running it again doesn't remove it, so must be throwed in by something

Reply:
ah, maybe we bumped yip, that uses herd now

Reply:
Ha, interesting, I would expect it to be marked as indirect. The more I know about go deps the less I understand them lol